JapaneseFilter/sections/specific.txt

```diff
@@ -721,6 +721,8 @@ e-rumoi.jp##.site_bottom_banner
 town.tobetsu.hokkaido.jp##.content6_wrap
 uub.jp###container > div.box
 portal.auone.jp##.news__item--ad
+portal.auone.jp##.news__items--rect-ad
+portal.auone.jp##.news__items--recommend-rect-ad-at
```

Comment:
Hm, I don't see this element.

Reply:
I am sorry I did not explain it enough.
It will show up when the UA to mobile.
It shows the same loading icon as in the screenshot.

Reply:
I see `##.news__items--rect-ad` on mobile, but don't `##.news__items--recommend-rect-ad-at` so far. Is this on the same page?

Reply:
yes,these are on the same page.
screenshot 1 is `##.news__items--rect-ad`
screenshot 2 is `##.news__items--recommend-rect-ad-at`

<details><summary>screenshot 1</summary>

![](https://i.slow.pics/ERObA0AR.png)

</details><br/>


<details><summary>screenshot 2</summary>

![](https://i.slow.pics/k6nAvWBD.png)

</details><br/>

Reply:
Btw, I don't see it on desktop, so it's better to move the rules to AdGuard Mobile.

Reply:
I forgot to move rules to mobile filter. 
sorry,I will fix later.